terminal/display_posix.go

```diff
@@ -0,0 +1,11 @@
+// +build !windows
+
+package terminal
+
+import (
+	"fmt"
+)
+
+func EraseInLine(mode EraseLineMode) {
```

Comment:
So this name is an artifact from `go-ansi` and I think it would probably be more appropriately named something like `EraseLine` - thoughts?